### DISCUSSION TITLE
Adding new DVD drives to Docker version of ARM

### DISCUSSION THREAD

**Discussion by HaroldRobi** [automatic-ripping-machine/automatic-ripping-machine#915] [Q&A]
I am running Docker arm version 2.6.30.  I attempted to add a new DVD to my server, I can see the server using lsscsi and lsblk but I can eject the disk from outside as well as inside the docker.  I scanned for new disks and it found one but ever time I insert a disk, ARM script never starts.

**Comment by microtechno9000** [comment]
can you post logs from the scan and your docker setup

**Reply by HaroldRobi** [reply]
Logs from my scan, where do I see those ... The Empty.log file is empty and the arm.log file is empty as well.  The ripping machine does work for /dev/sr0- /dev/sr2.  /dev/sr3 and /dev/sr4 are not opening the DVDs.

My Docker setup is as per the directions for setting up the docker.  

the base operating system is:
```
arm@ripasaurus-2:/etc$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
```

**Reply by HaroldRobi** [reply]
Please let me know what you need and where to get it from, and I will post it.

**Reply by HaroldRobi** [reply]
I would like to resolve this.  Do I need to update my docker image?

**Reply by HaroldRobi** [reply]
Any feed back from the devs?
